Source/ACE/Network/GameAction/GameActionType.cs

```diff
@@ -166,6 +166,9 @@ public enum GameActionType
 
         MovementEvent = 0xF74C,
 
+        // ACE ONLY GameActionEvent to send Events over the "Action" queue:
+        GameActionEvent = 0xF819, // Only used when checking the action queue for an event.
```

Comment:
Does this action code exist in the NetworkProtocol?  I don't see it in the (outdated) Skunkworks docs.  This is a pretty dangerous precedent.